### ISSUE TITLE
zenburn theme request

### ISSUE THREAD

**Issue by YuFei29** [NvChad/base46#191]
can we have my personal favorite theme zenburn？thanks!
[vim zenburn, original](https://github.com/jnurmine/Zenburn)
[emacs port](https://github.com/bbatsov/zenburn-emacs)
[nvim port](https://github.com/phha/zenburn.nvim)

**Comment by siduck** [comment]
will add this weekend :)

**Comment by tumes** [comment]
+1 for this, it seems like it hasn't made it in yet but I would absolutely love to get it in my setup at some point, it's a great theme.

**Comment by siduck** [comment]
@tumes  yes, i decided to port it in v3.0 which will probably release before the end of this year.

**Comment by tumes** [comment]
@siduck Did this happen to make it in under a different name? Regardless congrats on the release!

**Comment by siduck** [comment]
@tumes base46 v3.0 will take time! hopefully before next year

**Comment by tumes** [comment]
@siduck Oh shoot, my bad, I conflated this with the NvChad UI 3.0 release. Sorry for the pings, appreciate all your hard work!
